{
  "data_type": "CVE",
  "data_format": "MITRE",
  "data_version": "4.0",
  "CVE_data_meta": {
    "ID": "CVE-2021-44647",
    "ASSIGNER": "cve@mitre.org"
  },
  "problemtype": {
    "problemtype_data": [
      {
        "description": [
          {
            "lang": "en",
            "value": "CWE-843"
          }
        ]
      }
    ]
  },
  "references": {
    "reference_data": [
      {
        "url": "http://lua-users.org/lists/lua-l/2021-11/msg00204.html",
        "name": "http://lua-users.org/lists/lua-l/2021-11/msg00204.html",
        "refsource": "MISC",
        "tags": [
          "Mailing List",
          "Patch",
          "Vendor Advisory"
        ]
      },
      {
        "url": "http://lua-users.org/lists/lua-l/2021-11/msg00195.html",
        "name": "http://lua-users.org/lists/lua-l/2021-11/msg00195.html",
        "refsource": "MISC",
        "tags": [
          "Exploit",
          "Mailing List",
          "Vendor Advisory"
        ]
      },
      {
        "url": "https://access.redhat.com/security/cve/cve-2021-44647",
        "name": "https://access.redhat.com/security/cve/cve-2021-44647",
        "refsource": "MISC",
        "tags": [
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://security.gentoo.org/glsa/202305-23",
        "name": "GLSA-202305-23",
        "refsource": "GENTOO",
        "tags": []
      },
      {
        "url": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/P3EMGAQ5Y6GXJLY4K5DUOOEQT4MZ4J4F/",
        "name": "FEDORA-2022-473560d1a6",
        "refsource": "",
        "tags": []
      }
    ]
  },
  "description": {
    "description_data": [
      {
        "lang": "en",
        "value": "Lua v5.4.3 and above are affected by SEGV by type confusion in funcnamefromcode function in ldebug.c which can cause a local denial of service."
      }
    ]
  }
}